platooning_control/README.md

```diff
@@ -1,7 +1,7 @@
 # platoon_control
 
 Platooning Control plugin which allows platooning to maintain the gap; moreover, generates longitudinal and lateral control commands to follow the trajectory. The structure of this plugin is very similar to the control plugin for IHP2, so the same design document is included here.
-The difference between platoon_control, and platoon_control_ihp is that the IHP plugin includes logic to open or close the gap between platoon members, to allow for a new member to join or an existing memeber to exit the platoon. 
+The difference between platoon_control, and platoon_control_ihp is that the IHP plugin includes logic to open or close the gap between platoon members, to allow for a new member to join or an existing memeber to exit the platoon.
```

Comment:
I dont think we need this paragraph anymore since the control ihp will be removed. You can add a few sentences saying it includes logic from control ihp, and autoware pure pursuit